README.md

```diff
@@ -79,7 +79,7 @@ One of the primary advantage of CMake based build system is integration with oth
 1. Clone latest version:
 
   ```
-  git clone https://github.com/neuronsimulator/nrn
+  git clone --recursive https://github.com/neuronsimulator/nrn
```

Comment:
I believe the --recursive effect for the needed external submodules happens automatically via cmake when it is necessary (no need to get InterViews, CoreNEURON, NMODL, etc, when not needed)

Reply:
but how would it know? since only tell cmake whether want eg InterViews after doing the clone

Reply:
Cmake clones it into the nrn/external/interviews . Similarly with the others.